### PR TITLE
[1LP][RFR] Add a check to check if CU VM is running

### PR DESCRIPTION
**Pull request by nachandr**
{{pytest: cfme/tests/intelligence/reports/test_validate_chargeback_report.py --use-provider complete}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0b5f3f48902c6be282675995b643da03c90da650:

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :heavy_exclamation_mark: Line 589:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py [hunk 1]
```diff
@@ -64,10 +64,30 @@ def new_credential():
...
+            provider.mgmt.can_suspend and provider.mgmt.is_vm_suspended(vm_name) or \
+                provider.mgmt.can_pause and provider.mgmt.is_vm_paused(vm_name):
+                provider.mgmt.start_vm(vm_name)
+
+        logger.debug("Sleeping 15secs...(current state: {}, needed state: running)".format(
+            provider.mgmt.vm_status(vm_name)
+        ))
+        return False
+
+    wait_for(_cu_vm_running, num_sec=360, delay=15)
```
Comment (mkoura) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: This seems overkill to me, just
```
if not provider.mgmt.does_vm_exist(vm_name):
    pytest.skip("Skipping test, cu-24x7 VM does not exist")
if not provider.mgmt.is_vm_running(vm_name):
    provider.mgmt.start_vm(vm_name)
```
should be enough as ``provider.mgmt.start_vm`` does `wait_for` by itself.

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py [hunk 2]
```diff
@@ -64,10 +64,30 @@ def new_credential():
...
+
+    if not provider.mgmt.is_vm_running(vm_name):
+        pytest.skip("Skipping test, cu-24x7 VM is not running")
```
Comment (mkoura) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: Redundant, as ``wait_for`` or ``provider.mgmt.start_vm`` above will fail if it's not running.

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py (hunk 1 above)
Comment (nachandr) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: @mkoura , I've made the necessary code changes.

File: cfme/tests/intelligence/reports/test_validate_chargeback_report.py (hunk 2 above)
Comment (nachandr) [inline cfme/tests/intelligence/reports/test_validate_chargeback_report.py]: Taken care of this as well.

**Comment by nachandr** [comment]
The tests passed for v65 and v6, but failed on RHEVM36 on PRT.The failure is not related to the code changes made through this PR.I'll investigate the cause and fix it through another PR.

**Pull request merged**